apps/wa/wa-task-batch-reconfiguration/aat-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-reconfiguration
+spec:
+  values:
+    job:
+      schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
```

Comment:
Reconfiguration batch job is expected to run every 15 mins on prod.
For testing on aat and other test env, we would like it run every 2 mins, hence the split between clusters

Reply:
What happens if it fails/ doesnt finish it in 15 mins and other cluster start running the job ?

Reply:
If reconfiguration batch job cannot lock any task for reconfigure, it will report in logs and job completes with success.
Any such failed tasks will get picked in the next run.